snappy/binaries.go

```diff
@@ -77,7 +77,6 @@ export HOME="$SNAP_USER_DATA"
 
 # export old pwd
 export SNAP_OLD_PWD="$(pwd)"
```

Comment:
The old working directory doesn't need to be stored now if we're not changing it?